src/App.css

```diff
@@ -1,3 +1,310 @@
 .App {
   font-family: Arial, Helvetica, sans-serif;
 }
+
+.navBar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 10px;
+  background-color: #fff;
+}
```

Comment:
- [ ] As mentioned by the previous reviewer, let's use the correct font here and the user icon, on the right, should be made smaller and placed on the extreme right. The goal here is to match the Zeplin design as much as possible. Cheers 👍 

Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/202866451-c4f640f2-dda1-4cc5-962c-063a75af0089.png)  | ![image](https://user-images.githubusercontent.com/60618877/202866429-b3d36d33-be5a-448f-822f-bd7405f7e790.png)